### ISSUE TITLE
Reporting - Sometimes an unwanted page break appears at the beginning of the Questionnaires

### ISSUE THREAD

**Issue by jocelyncardinal** [CAUCA-9-1-1/survip#546]
**Describe the bug**
Reporting - Sometimes an unwanted page break appears at the beginning of the Questionnaires

![Capture d’écran, le 2019-05-14 à 11.11.34.png](https://images.zenhubusercontent.com/5aeb25264b5806bc2bd1a294/cd1815a6-51b8-4e39-b8c0-35cc02f77411)

**To Reproduce**
It happen something but I don't know yet how to reproduce it. More to come ...

**Comment by iamkinetic** [comment]
Well since page break is not an html thing and we never add any page break, i would think the Html to Pdf tools is trying to keep the table's row on one page.

**Comment by iamkinetic** [comment]
Well it was a known issue in some earlier version, i don't know which version we are currently using.

https://stackoverflow.com/questions/13516534/how-to-avoid-page-break-inside-table-row-for-wkhtmltopdf

There's some css stuff I could try.

**Comment by iamkinetic** [comment]
css trick (add thead and tbody) :

```
tr, td div, th div
    page-break-inside: avoid;
}
```

**Comment by jocelyncardinal** [comment]
WOW!

**Comment by iamkinetic** [comment]
Je vais injecter ce css à la création du document, ça devrait régler le problème.

**Comment by iamkinetic** [comment]
@Jeepsboucher SVP indiquer ce qui fait que ce point est bloqué et ce que ça prend pour le débloquer.

**Comment by Jeepsboucher** [comment]
@iamkinetic J'ai essayer le code que tu avais mis plus haut, cela n'a pas fonctionné. J'ai essayer de trouver une autre solution, mais au final rien n'a marché. De plus, certain rapports n'ont pas le problème.

**Comment by jocelyncardinal** [comment]
Effectivement, le probleme n'arrive pas sur tout les rapports mais de memoire il me semble que ca arrive quand meme assez régulièrement. Est-ce que ca dépend des champs (questions) qui ont été rempli ou laissé vide???

**Comment by iamkinetic** [comment]
Nous n'avons pas réussi à régler ce problème et nous le mettons de côté pour l'instant.

**Comment by jocelyncardinal** [comment]
Ce n’est pas une issue bloquante. On va voir à l’utilisation si on peut trouver le moyen de reproduire le problème de façon systématique. Ensuite, ça sera plus facile a réglé ou trouver un work-around
